src/python_testing/matter_testing_infrastructure/chip/testing/matter_testing.py

```diff
@@ -205,7 +205,7 @@ def utc_datetime_from_posix_time_ms(posix_time_ms: int) -> datetime:
     return datetime.fromtimestamp(seconds, timezone.utc) + timedelta(milliseconds=millis)
 
 
-def compare_time(received: int, offset: timedelta = timedelta(), utc: int = None, tolerance: timedelta = timedelta(seconds=5)) -> None:
+def compare_time(received: int, offset: timedelta = timedelta(), utc: typing.Optional[int] = None, tolerance: timedelta = timedelta(seconds=5)) -> None:
```

Comment:
nit
but do you think `from typing import Optional` and just putting `Optional` directly will make the code more readable??

Reply:
It's actually already imported like that - I've swapped now to just using Optional.